tests/test_technology.py

```diff
@@ -319,19 +320,19 @@ def test_co2_rate(advanced_tech):
     assert advanced_tech.co2_rate.value == 10.0
     assert advanced_tech.co2_rate.units == expected_unit
 
-    advanced_tech.co2_rate = float_val * kg / other_energy_unyt
+    advanced_tech.co2_rate = float_val * megatonnes / other_energy_unyt
     assert advanced_tech.co2_rate.value == pytest.approx(
         3412141.5, 0.5)
     assert advanced_tech.co2_rate.units == expected_unit
 
     advanced_tech.unit_power = "kW"
     advanced_tech.unit_time = "day"
-    assert advanced_tech.co2_rate.units == kg*(kW * day)**-1
+    assert advanced_tech.co2_rate.units == megatonnes*(kW * day)**-1
 
     advanced_tech.unit_power = "GW"
     advanced_tech.unit_time = "hr"
     advanced_tech.unit_mass = "ton"
-    advanced_tech.co2_rate = 10.0 * kg*(kW*hr)**-1
+    advanced_tech.co2_rate = 1e-2 * megatonnes*(GW*hr)**-1
```

Comment:
Consider adding a variable to take `1e-2` add naming it based on the conversion it comes from.